win32ctypes/core/cffi/_kernel32.py

```diff
@@ -66,6 +74,18 @@ def _GetACP():
     return kernel32.GetACP()
 
 
+def _GetWindowsDirectory():
+    buffer = ffi.new('wchar_t[256]')
```

Comment:
That was my concern also. However, MAX_PATH has been the same value for 20 years, I suspect it isn't changing (and, I got it wrong -- it's actually 260).